src/ResourceManager/OperationalInsights/Commands.OperationalInsights/Query/InvokeOperationalInsightsQuery.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.Azure.Commands.Common.Authentication;
+using Microsoft.Azure.Commands.OperationalInsights.Models;
```

Comment:
Please add help for this cmdlet: https://github.com/Azure/azure-powershell/wiki/Azure-Powershell-Developer-Guide#adding-help-content

Reply:
Fixed.